src/pages/home/report/ReportActionsView.js

```diff
@@ -219,9 +209,11 @@ class ReportActionsView extends React.Component {
 
     /**
      * Updates and sorts the report actions by sequence number
+     *
+     * @param {Array<{sequenceNumber, actionName}>} reportActions
      */
-    updateSortedReportActions() {
-        this.sortedReportActions = _.chain(this.props.reportActions)
+    updateSortedReportActions(reportActions) {
+        this.sortedReportActions = _.chain(reportActions)
```

Comment:
Updated usages so this doesn't get called for each render but only when there's new data